addons/l10n_eg_edi_eta/models/res_currency_rate.py

```diff
@@ -11,7 +12,10 @@ class ResCurrencyRate(models.Model):
     @api.onchange('company_rate')
     def _onchange_rate_warning(self):
         # We send the ETA a rate that is 5 decimal accuracy, so to ensure consistency, Odoo should also operate with 5 decimal accuracy rate
-        if self.company_id.account_fiscal_country_id.code == 'EG' and self.inverse_company_rate != round(self.inverse_company_rate, 5):
+        if (
+            self.company_id.account_fiscal_country_id.code == 'EG' and
+            float_compare(self.inverse_company_rate, round(self.inverse_company_rate, 5), precision_digits=10) != 0
```

Comment:
not sure the != 0 is useful but since it's a backport i think we can let it like this :smile: